qemu/tests/virtio_fs_support_win_fs.py

```diff
@@ -0,0 +1,96 @@
+    session = utils_test.qemu.setup_win_driver_verifier(session,
+                                                        driver_verifier,
+                                                        vm)
+    error_context.context("Create the viofs service.", test.log.info)
+    virtio_fs_utils.create_viofs_service(test, params, session)
+    error_context.context("Add NTFS filesystem to virtiofs.", test.log.info)
+    reg_add_cmd = params.get("reg_add_cmd")
+    session.cmd(reg_add_cmd)
+    error_context.context("restart virtiofs service to make the registry "
+                          "change work.", test.log.info)
+    virtio_fs_utils.stop_viofs_service(test, params, session)
+    virtio_fs_utils.start_viofs_service(test, params, session)
+
+    winutils_driver_letter = utils_misc.get_winutils_vol(session)
+    shared_driver_letter = virtio_fs_utils.get_virtiofs_driver_letter(test,
+                                                                      fs_target,
+                                                                      session)
+    winutils_pack_path = winutils_driver_letter + winutils_pack_path
+    autoIt_path = winutils_driver_letter + autoIt_path
+    script_path = winutils_driver_letter + script_path
+    copy_cmd = "xcopy %s %s:\\ /Y" % (winutils_pack_path, shared_driver_letter)
+    error_context.context("Copy the executable to shared dir.",
+                          test.log.info)
+    session.cmd(copy_cmd)
+
+    error_context.context("Create the new user", test.log.info)
+    session.cmd(params.get("add_user_cmd"))
+    error_context.context("Replace the default username and password with the "
+                          "new user.", test.log.info)
+    session.cmd(reg_add_username)
+    session.cmd(reg_add_pwd)
+    try:
+        session.cmd("taskkill /im rss6.exe /f && taskkill /im rss.exe /f"
+                    " && shutdown -r")
+    except ShellProcessTerminatedError:
+        test.log.info("Once the rss.exe is killed, the "
+                      "ShellProcessTerminatedError will be raised. This "
+                      "exception is expected and is NOT considered as an error"
+                      ".")
+    finally:
+        error_context.context("Login the guest with the new user.",
+                              test.log.info)
+        session = vm.wait_for_login(timeout=500, internal_timeout=120,
+                                    serial=True)
+    result_on_shared = "%s:\\result.txt" % shared_driver_letter
+    error_context.context("Run autoit script to install executable in "
+                          "explorer.", test.log.info)
+    session.cmd("start /w " + autoIt_path + " " + script_path + " " +
+                result_on_shared + " " + shared_driver_letter + ":")
+    output = session.cmd_output("type " + result_on_shared)
+    test.log.info("The title and text of the window are: %s" % output)
+    if "7-Zip" not in output and "Setup" not in output:
```

Comment:
How is the case result with only `if "7-Zip" not in output:`? pass or failed?
CC @xiagao 
I mean if we will miss the hole of testing with the current condition `if "7-Zip" not in output and "Setup" not in output:`?
I think `if "7-Zip" not in output or "Setup" not in output:` may cover all.